publish-externals/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import path from "path"
+import {hashnodePostHandler} from "./utils/hashnode"
+import {extractFrontMatterAndContent} from "./utils/markdown"
+import {devtoPostHandler} from "./utils/devto"
+
+const main = async () => {
+  const changedFiles = process.argv[2].split(" ")
+  for (const file of changedFiles) {
+    if (file.startsWith("blog/")) {
+      try {
+        const filePath = path.join(__dirname, "../../", file)
+
+        const {frontMatter, content} = extractFrontMatterAndContent(filePath)
+        //call both and execute simultaneously
+        await Promise.all([hashnodePostHandler(frontMatter, content), devtoPostHandler(frontMatter, content)])
+      } catch (error) {
```

Comment:
We should add some logs here to see what succeed and what's pending.

Reply:
Do you mean in specific like searching for articles, uploading etc. ? As it already logs the general info.

Reply:
Internal logs aren't needed as much as over here.

Reply:
I still don't get it, can you give more specific details about what is required to do ? Do you want me to log the processes in the main scope instead of the handler function ?